hadoop-hdds/rocksdb-checkpoint-differ/src/test/java/org/apache/ozone/rocksdb/util/TestManagedSstFileReader.java

```diff
@@ -53,7 +54,8 @@
 /**
  * ManagedSstFileReader tests.
  */
-public class TestManagedSstFileReader {
+@Native(ROCKS_TOOLS_NATIVE_LIBRARY_NAME)
```

Comment:
The annotation shouldn't be here I believe this test should just use the jar either from the apache maven repo or the one built in this run. I believe we shouldn't tag this test.

Reply:
> this test should just use the jar either from the apache maven repo or the one built in this run

The point of the change is to avoid building the native library for all checks except the `native` one.  The `@Native` tag is added to ensure that this test is run in the `native` check and no other checks.  All test classes with `loadLibrary(ROCKS_TOOLS_NATIVE_LIBRARY_NAME)` have to be tagged.

Reply:
Say if changes happen to ManagedSSTFileReader and not the hdds-rocks-native module directory. Will this particular test class run or not?